boot/crio.sh

```diff
@@ -11,6 +11,16 @@ mkdir -p $XDG_DATA_HOME/containers/oci/hooks.d $XDG_CONFIG_HOME/containers $XDG_
 if [[ ! -f $XDG_CONFIG_HOME/crio/crio.conf ]]; then
 	cat >$XDG_CONFIG_HOME/crio/crio.conf <<EOF
 registries = ['registry.access.redhat.com', 'registry.fedoraproject.org', 'docker.io']
+[crio.runtime]
+  default_runtime = "crun"
+  [crio.runtime.runtimes]
+    [crio.runtime.runtimes.crun]
+      runtime_path = "$U7S_BASE_DIR/bin/crun"
+      runtime_root = "$XDG_RUNTIME_DIR/crio/crun"
+# override built-in "runc" config to avoid error: 'runtime validation: \"runc\" not found in $PATH'
```

Comment:
@giuseppe Do you know the solution for this? Setting `default_runtime = "crun"` doesn't solve.

Reply:
Opened https://github.com/cri-o/cri-o/issues/3360 for asking